### PR TITLE
8345876: Update nativeAddAtIndex comment to match the code

### PR DESCRIPTION
**Pull request by saagarjha**
---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8345876](https://bugs.openjdk.org/browse/JDK-8345876): Update nativeAddAtIndex comment to match the code (**Bug** - P4)

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**) Review applies to [62da15c0](https://git.openjdk.org/jdk/pull/22627/files/62da15c050e4b2b7ed783e121bdd58efb40176a2)
 * [Alexander Zvegintsev](https://openjdk.org/census#azvegint) (@azvegint - **Reviewer**)
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**) Review applies to [62da15c0](https://git.openjdk.org/jdk/pull/22627/files/62da15c050e4b2b7ed783e121bdd58efb40176a2)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/22627/head:pull/22627` \
`$ git checkout pull/22627`

Update a local copy of the PR: \
`$ git checkout pull/22627` \
`$ git pull https://git.openjdk.org/jdk.git pull/22627/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 22627`

View PR using the GUI difftool: \
`$ git pr show -t 22627`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/22627.diff">https://git.openjdk.org/jdk/pull/22627.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/22627#issuecomment-2531491042)
</details>

### CODE REVIEW

**Comment by prsadhuk** [comment]
No issue listed..I have added one

**Comment by saagarjha** [comment]
Ok so I think that solved the "I don't have a way to make a bug for this" issue. I have signed an OCA a while back, is there anything else I need to do there?

**Review by prsadhuk — APPROVED** [review]

**Review by azvegint — APPROVED** [review]
> Ok so I think that solved the "I don't have a way to make a bug for this" issue. I have signed an OCA a while back, is there anything else I need to do there?


It looks like you are good to go, just follow the last paragraph of the https://github.com/openjdk/jdk/pull/22627#issuecomment-2524872192

**Review by azuev-java — APPROVED** [review]

File: src/java.desktop/macosx/native/libawt_lwawt/awt/CMenuBar.m
Comment (aivanov-jdk) [inline src/java.desktop/macosx/native/libawt_lwawt/awt/CMenuBar.m:1]: Please update the copyright line:

```diff
- * Copyright (c) 2011, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved. 
```

**Review by aivanov-jdk — APPROVED** [review]

**Comment by saagarjha** [comment]
Oops I guess I wasn't supposed to rebase on top of master. Sorry :(

Anyway, I added the copyright year in. I'll hold off on `/integrate` just in case I'm supposed to get this reviewed again, but if it looks good I can go ahead and do that.

**Review by azvegint — APPROVED** [review]

**Comment by aivanov-jdk** [comment]
> Oops I guess I wasn't supposed to rebase on top of master. Sorry :(

You must never use `git rebase` for a branch being reviewed. All your commits get new hashes as the result of the rebase operation, therefore all the comments become orphaned.

The correct way is to use `git merge master -m 'Merge master'` after you pulled all the latest changes for master.

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**